WINDOWS.md

```diff
@@ -0,0 +1,56 @@
+# Cage on Windows
+
+Generally speaking, docker-compose and docker engine are a work in progress on
+Windows. This document should give you all the information you need on
+available builds and overall guidance when working with cage
+on Windows.
+
+## Setup
+
+First make sure you install rust for x86_64-pc-windows-gnu using the
+`rustup-init` installer from [https://www.rustup.rs/](https://www.rustup.rs/).
+You should make sure you use either `stable` or `beta` toolchains.
+
+Next up you'll need to get a build env going with Msys2. Grab a mingw installer
+from [Mingw-w64's sf.net](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/)
```

Comment:
Is there an official, non-Source Forge site for this? Source Forge has had issues with installing third-party toolbars and other such nonsense. If not, no worries.